homeassistant/components/whirlpool/entity.py

```diff
@@ -19,8 +19,9 @@ def __init__(self, appliance: Appliance, unique_id_suffix: str = "") -> None:
 
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, appliance.said)},
-            name=appliance.name.capitalize(),
+            name=appliance.name.capitalize() if appliance.name else appliance.said,
             manufacturer="Whirlpool",
+            model=appliance.appliance_info.model_number,
```

Comment:
is this model or model_id

Reply:
its something like SPIW312A2WF. changed to model_id